VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls-base.component.ts

```diff
@@ -386,4 +403,27 @@ export abstract class HearingControlsBaseComponent implements OnInit, OnDestroy
     changeDeviceSelected() {
         this.changeDeviceToggle.emit();
     }
+
+    isAnotherHostPresent(participants: ParticipantModel[]): boolean {
+        let isAnotherHostInHearing = false;
+
+        const hosts = participants.filter(
+            x =>
+                x.id !== this.participant.id &&
+                (x.role === Role.Judge || x.role === Role.StaffMember || x.role === Role.JudicialOfficeHolder)
+        );
+
+        if (hosts.length === 0) {
+            return isAnotherHostInHearing;
+        }
+
+        hosts.forEach(host => {
+            if (host.status === ParticipantStatus.InHearing) {
+                isAnotherHostInHearing = true;
+                return;
+            }
+        });
```

Comment:
Use any. Something like:
```suggestion
   if(hosts.any(host => host.status === ParticipantStatus.InHearing) {
        return true;
   }
```